addons/microsoft_calendar/models/calendar.py

```diff
@@ -127,13 +127,14 @@ def _get_microsoft_sync_domain(self):
         day_range = int(ICP.get_param('microsoft_calendar.sync.range_days', default=365))
         lower_bound = fields.Datetime.subtract(fields.Datetime.now(), days=day_range)
         upper_bound = fields.Datetime.add(fields.Datetime.now(), days=day_range)
-        return [
+        domain = [
             ('partner_ids.user_ids', 'in', self.env.user.id),
             ('stop', '>', lower_bound),
             ('start', '<', upper_bound),
             # Do not sync events that follow the recurrence, they are already synced at recurrence creation
             '!', '&', '&', ('recurrency', '=', True), ('recurrence_id', '!=', False), ('follow_recurrence', '=', True)
         ]
+        return self._get_microsoft_domain(domain)
```

Comment:
From a functional point of view, it seems OK but looking at the code is confusing.
We have a `_get_microsoft_sync_domain` that calls a `_get_microsoft_domain` method. It is confusing. Could you find a better naming for the new method? 


Before your change, _get_microsoft_sync_domain was not always calling these conditions when no fullsync:
```
is_active_clause = (self._active_name, '=', True) if self._active_name else expression.TRUE_LEAF
            domain = expression.AND([domain, [
                '|',
                '&', ('ms_universal_event_id', '=', False), is_active_clause,
                ('need_sync_m', '=', True),
            ]])
```
Are you sure it won't cause any harm to call these conditions for calendar events ?

Reply:
Thanks for the review,
I changed the function's name to `_extend_microsoft_domain`.

Also, I changed the `full_sync_m` default value to `True`, and removed the `with_context` in other places, and only kept it in the `_get_microsoft_records_to_sync`.
So, if the function would be called without adding context it won't return that additional domain.